python/python/lance/arrow.py

```diff
@@ -169,13 +178,16 @@ class ImageURIArray(ImageArray):
     """
 
     @classmethod
-    def from_uris(cls, uris: Union[pa.StringArray, Iterable[Union[str, Path]]]):
+    def from_uris(
+        cls,
+        uris: Union[pa.StringArray, pa.LargeStringArray, Iterable[Union[str, Path]]],
+    ):
         """
-        Create an ImageURIArray from a pa.StringArray or an iterable.
+        Create an ImageURIArray from an array or sequence of URIs.
```

Comment:
This seems less precise.  What's a "sequence"?  Is it the same thing as an iterable?

Reply:
Fair. Changed back to saying iterable.